resources/views/index.edge

```diff
@@ -24,8 +24,9 @@
         <div data-type='medicine'><table><tr><td>I need medicine</td></tr></table></div><div data-type='food'><table><tr><td>I need food</td></tr></table></div><div data-type='supplies'><table><tr><td>I need supplies</td></tr></table></div>
       </needs>
       <form>
+        <destroyed><div>:-(</div>We couldn't determine your location!</destroyed>
         <inputwrap><input type='text'><fx1></fx1><fx2></fx2><placeholder>My name</placeholder></inputwrap>
-        <inputwrap><input type='text'><fx1></fx1><fx2></fx2><placeholder>My location</placeholder></inputwrap>
+        <inputwrap class='location locating'><span>My current location</span><placeholder class='busy'>Locating</placeholder><placeholder class='done'>Location</placeholder></inputwrap>
```

Comment:
@vartiainen I think "Location" in done case should be "Current Location" ?

Reply:
ok, now it's "Current Location"